android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -571,17 +587,43 @@ public void startShareScreen(final Promise promise) {
       return;
     }
 
+    if (customizedMeetingUIEnabled) {
+      final MediaProjectionManager manager =
+        (MediaProjectionManager) getReactApplicationContext().getSystemService(Context.MEDIA_PROJECTION_SERVICE);
+
+      if (manager != null) {
+        Intent intent = manager.createScreenCaptureIntent();
+
+        reactContext.getCurrentActivity().startActivityForResult(intent, 99);
+      }
+
+      promise.resolve(null);
+    } else {
+      final InMeetingShareController shareController = zoomSDK.getInMeetingService().getInMeetingShareController();
+
+      MobileRTCSDKError result = shareController.startShareScreenContent();
+
+      if (result == MobileRTCSDKError.SDKERR_SUCCESS) {
+        promise.resolve(null);
+      } else {
+        promise.reject("ERR_ZOOM_MEETING_CONTROL", "Start share screen error, status: " + result.name());
+      }
+    }
+  }
+
+  private void startZoomScreenShare(Intent intent) {
     UiThreadUtil.runOnUiThread(new Runnable() {
       @Override
       public void run() {
+        final ZoomSDK zoomSDK = ZoomSDK.getInstance();
         final InMeetingShareController shareController = zoomSDK.getInMeetingService().getInMeetingShareController();
 
-        MobileRTCSDKError result = shareController.startShareScreenContent();
+        MobileRTCSDKError result = shareController.startShareScreenSession(intent);
 
         if (result == MobileRTCSDKError.SDKERR_SUCCESS) {
-          promise.resolve(null);
+          sendEvent("MeetingEvent", "screenShareSuccess");
         } else {
-          promise.reject("ERR_ZOOM_MEETING_CONTROL", "Start share screen error, status: " + result.name());
+          sendEvent("MeetingEvent", "screenShareError:" + result.name());
```

Comment:
Make it one error with additional `error` param
```suggestion
          sendEvent("MeetingEvent", "screenShareError:", result);
```

This will need implementation of:
```java
private void sendEvent(String name, String event, MobileRTCSDKError error) {
```